### ISSUE TITLE
About renaming of  apache-shardingsphere-operator-cluster-charts 

### ISSUE THREAD

**Issue by xuanyuan300** [apache/shardingsphere-on-cloud#160]
Currently, we have apache-shardingsphere-operator-charts and apache-shardingsphere-operator-cluster-charts  charts, but the both name are  confusing.
Refered to the installation [rabbitmq-cluster-operator](https://github.com/bitnami/charts/tree/main/bitnami/rabbitmq-cluster-operator) and [grafana-operator](https://github.com/bitnami/charts/tree/main/bitnami/grafana-operator) charts, we plan merge apache-shardingsphere-operator-charts and apache-shardingsphere-operator-cluster-charts, the name of new chart will be renamed to  `apache-shardingsphere-cluster-operator-charts`. The new chart will  have both  shardingsphere-operator and shardingsphere-proxy-cluster installed.

Also, the pod name installed by operator is too long, the following is example:
```bash
ss-apache-shardingsphere-proxy-cluster-57999bd6b8-fzcdx           1/1     Running   0          21h
ss-apache-shardingsphere-proxy-cluster-57999bd6b8-ms2wb           1/1     Running   0          21h
ss-apache-shardingsphere-proxy-cluster-57999bd6b8-r9kq9           1/1     Running   0          21h
ss-apache-shardingsphere-proxy-cluster-operator-577dc4fdc9p5k2r   1/1     Running   0          21h
ss-apache-shardingsphere-proxy-cluster-operator-577dc4fdc9qxb7m   1/1     Running   0          21h
```

`ss` is `RELEASE_NAME` for helm.
`apache-shardingsphere-proxy-cluster` is the default value of the `nameOverride` for our chart.

When users set `RELEASE_NAME`, it is easy to exceed the 63 characters of the kubernetes object name character limit,
please refer to `https://kubernetes.io/docs/concepts/overview/working-with-objects/names/`, so maybe we need to simplify the value of `nameOverride`.

**Comment by mlycore** [comment]
Maybe we could use `shardingsphere-proxy` for ShardingSphere itself and use `shardingsphere-operator` for operator ?.

Both packaged in the `apache-shardingsphere-operator-charts` ?

**Comment by xuanyuan300** [comment]
> Maybe we could use for ShardingSphere itself and use for operator ?.`shardingsphere-proxy``shardingsphere-operator`
> 
> Both packaged in the ?`apache-shardingsphere-operator-charts`

I agree with you.

**Comment by mlycore** [comment]
fixed in #162
